AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreen.swift

```diff
@@ -39,7 +39,10 @@ struct CompleteScreen: View {
             if viewModel.isMapSelected {
                 SearchCompleteScreenMapView(longitude: viewModel.sessionLongitude, latitude: viewModel.sessionLatitude)
             } else {
-                SearchAndFollowChartView(viewModel: viewModel.chartViewModel)
+                SearchAndFollowChartView(viewModel: viewModel.chartViewModel, streamID: $viewModel.selectedStream)
```

Comment:
Why do we need that binding for streamID? And do we need to add this `onStreamChange` method at all? Maybe we could just call `generateEntries` function on chartViewModel from the ComleteScreenViewModel instead?

Reply:
You could add a code
```
if let stream = externalSessionWithStreams?.streams.first(where:  { $0.id == id }) {
            defineChartRange(with: stream)
        }
 ``` 

in `selectedStream(with id: Int)` function for example

Reply:
And add `entries = []` in generateEntries method

Reply:
Wow, yes. thaaaanks!